src/Components/CreateImageWizard/formComponents/Packages.js

```diff
@@ -156,60 +163,99 @@ const Packages = ({ defaultArch, ...props }) => {
         };
     });
 
-    // move selected packages
-    const moveSelected = (fromAvailable) => {
-        const sourcePackages = fromAvailable ? packagesAvailable : packagesChosen;
-        const destinationPackages = fromAvailable ? packagesChosen : packagesAvailable;
+    const areFound = (filter, packageList) => {
+        return filter === undefined ? true :
```

Comment:
No need to change this. Just in general I think that once inline conditionals start getting chained together they get hard to read. I would probably use an if else here or add a comment to improve the readability. But, this is correct and no need to change it.